### PR TITLE
AHMET PR 2

### PR DESCRIPTION
**Pull request by tech-extremists**

### CODE REVIEW

File: math.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class math {
+    public static int add(int num1, int num2) {
+        int answer = 0;
+        int answer = num1 + num2;
+        return ans;
+    }
+}
```
Comment (tech-extremists) [inline math.java:7]: !suggest add comment

Comment (tech-extremists) [inline math.java:7]: !suggest add comment

Comment (tech-extremists) [inline math.java:7]: !suggest add comments to the function please!

Comment (tech-extremists) [inline math.java:7]: !suggest add comments please!

File: math.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+public class math {
+    public static int add(int num1, int num2) {
+        int answer = 0;
+        int answer = num1 + num2;
+        return ans;
```
Comment (tech-extremists) [inline math.java:5]: !suggest add comments

File: math.java (hunk 1 above)
Comment (tech-extremists) [inline math.java:7]: !suggest add comments to the code please.

Comment (tech-extremists) [inline math.java:7]: !suggest add comments

Comment (tech-extremists) [inline math.java:7]: !suggest change name of num1 and num2

File: controller.java [hunk 3]
```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
...
 
-        // Perform addition
-        int sum = num1 + num2;
-
+int summation = math.add(number2, num1);
+        int summation = math.add(num2, num1);
+        System.out.println("xofsz");
         // Print the result
-        System.out.println("The sum of " + num1 + " and " + num2 + " is: " + sum);
+        System.out.println("The result of " + num1 + " and " + num2 + " is: " + sum);
```
Comment (tech-extremists) [inline controller.java:12]: !suggest change num1 to number2

File: math.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    }
```
Comment (tech-extremists) [inline math.java:6]: !suggest add comments

Comment (tech-extremists) [inline math.java:6]: !suggest add comments to the code

File: .github/workflows/pr_agent.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review]
```
Comment (tech-extremists) [inline .github/workflows/pr_agent.yml:3]: !suggest remove this

File: .github/workflows/pr_agent.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+on:
+  pull_request:
```
Comment (tech-extremists) [inline .github/workflows/pr_agent.yml:2]: !suggest remove this

File: controller.java [hunk 7]
```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
         int num1 = 10;
         int num2 = 20;
 
-        // Perform addition
-        int sum = num1 + num2;
```
Comment (tech-extremists) [inline controller.java:9]: !suggest change num1 to number1

File: controller.java [hunk 8]
```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
         int num1 = 10;
         int num2 = 20;
```
Comment (ali-tunahan) [inline controller.java:6]: !suggestion refactor num1 and num2 to number1 and number2 across the file

File: math.java (hunk 4 above)
Comment (ali-tunahan) [inline math.java:6]: !suggest first declaration of answer should be removed. This causes an exception!

File: controller.java [hunk 9]
```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
...
 
-        // Perform addition
-        int sum = num1 + num2;
-
+int summation = math.add(number2, num1);
```
Comment (ali-tunahan) [inline controller.java:8]: !suggestion rename the variable to sum

Comment (ali-tunahan) [inline controller.java:8]: !suggest rename the variable to sum

Comment (ali-tunahan) [inline controller.java:8]: !suggest rename the variable to sum

**Pull request closed without merging**